ci/create-windows-image.ps1

```diff
@@ -13,13 +13,14 @@
 
 # https://chocolatey.org/docs/how-to-setup-offline-installation
 $CHOCOLATEY_VERSION="0.10.11"
+$DOTNETCORE_SDK_VERSION="2.1.302"
 $GIT_VERSION="2.18.0"
 $RESHARPER_CLI_VERSION="2018.1.2"
 $SPATIAL_VERSION="1.1.9"
 
 # From https://public-cdn.cloud.unity3d.com/hub/prod/releases-win32.json
-$UNITY_VERSION="2018.2.0b10"
-$UNITY_RELEASE_HASH="4bc57476174c"
+$UNITY_VERSION="2018.2.1f1"
+$UNITY_RELEASE_HASH="1a9968d9f99c"
```

Comment:
these variables are stored in 
`workers/unity/ProejctSettings/ProjectVersion.txt`
and
`ci/UnityEditorHash.txt`

Reply:
Current master is also not on 2018.2.1f1, but on 2018.2.0f2